### ISSUE TITLE
Improve documentation for custom widgets and widget customization

### ISSUE THREAD

**Issue by alvarolm** [cogentcore/core#1082]
### Describe the feature

I believe that proactively fomenting the users to write or customise their own widgets its a key feature(s) for usability and adoption of the framework, this would provide diversity of components over time and other benefits.

Beautiful work !

**Comment by kkoreilly** [comment]
Yes, we have strong support for custom widgets and widget customization, and we are planning to write more documentation on that soon. For now, there is documentation on the [advanced updating](https://www.cogentcore.org/core/advanced/updating) page.

**Comment by kkoreilly** [comment]
When I do this, I will also clean up the advanced updating page and maybe move some of it to other pages like an advanced plans page.

I will also add documentation for custom `core.Value` widgets and note that you need to use `tree.Add*` for adding children to them in their `Init` functions.
